From: Michal Privoznik <mprivozn@redhat.com>
Subject: [PATCH] Fix apparmor meson.build path for virt-aa-helper

The profile template was renamed to usr.libexec but meson.build still
references the old usr.lib path.

--- a/src/security/apparmor/meson.build
+++ b/src/security/apparmor/meson.build
@@ -1,5 +1,5 @@
 apparmor_gen_profiles = [
-  'usr.lib.libvirt.virt-aa-helper',
+  'usr.libexec.libvirt.virt-aa-helper',
   'usr.sbin.virtqemud',
   'usr.sbin.virtxend',
 ]
